### PR TITLE
Fix/filter recipes

### PR DESCRIPTION
**Pull request by KyleWong2510**
Type of change made:

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

Detailed Description:
Paired with Becca to make filter function more dynamic to run through different arrays.

Why is this change required? What problem does it solve?
Page was changing when filters were being applied to my recipes page

Were there any challenges that arose while implementing this feature? If so, how were they addressed?
Finding where to make changes to the array

Files modified:
- [ ] index.html
- [ ] styles.css
- [x] scripts.js

Other files:

### CODE REVIEW

**Comment by KyleWong2510** [comment]
Need to add corbin's 'recipesToCook' functionality to the filter function

File: src/scripts.js [hunk 1]
```diff
@@ -174,35 +174,65 @@ function capitalize(words) {
...
 }
 
-function findCheckedBoxes() {
+function filterRecipesOnPage() {
+  if (document.querySelector('.welcome-msg').style.display !== 'none') {
+    findCheckedBoxes(allRecipes)
+  }
+  if (document.querySelector(".my-recipes-banner").style.display !== 'none') {
+    findCheckedBoxes(user.favoriteRecipes)
+  }
```
Comment (b-stein) [inline src/scripts.js:183]: great logic here observing the toggling CSS!

**Comment by b-stein** [comment]
Good work! Maybe later on we can refactor this functionality to make the code shorter, but awesome logic & bug fix 👊

**Comment by MarchCorbin** [comment]
Nice work getting the filter working!

**Pull request merged**